fs/rc/webgui/plugins.go

```diff
@@ -65,14 +67,16 @@ var (
 )
 
 func init() {
-	cachePath = filepath.Join(config.CacheDir, "webgui")
-	PluginsPath = filepath.Join(cachePath, "plugins")
-	pluginsConfigPath = filepath.Join(PluginsPath, "config")
+	if rcflags.Opt.WebUI {
```

Comment:
I'm not sure this will work as I don't think `rcflags.Opt` will have been initialised when the `init()` method is called. Did you try it and see what happened?

Reply:
Ahh, yes. After I removed the directories and tried again, it is not working. 
Any possible workaround for this?

Reply:
Should we have a different flag for enabling plugins? As during testing, we don't want the webui to be instantiated, but the plugins should be tested.

Reply:
I don't think you can read flags at that point... You can use a `sync.Once` to initialise it elsewhere if you want?